hydra-plutus/src/Hydra/Contract/Initial.hs

```diff
@@ -125,15 +127,17 @@ checkCommit commitValidator headId committedRef context =
     maybe mempty (txOutValue . txInInfoResolved) $ findOwnInput context
 
   committedValue =
-    maybe mempty (txOutValue . snd) committedTxOut
+    foldMap (txOutValue . txInInfoResolved) committedUTxO
 
-  committedTxOut = do
-    ref <- committedRef
-    (ref,) . txInInfoResolved <$> findTxInByTxOutRef ref txInfo
+  committedUTxO = do
+    flip fmap committedRefs $ \ref ->
+      case findTxInByTxOutRef ref txInfo of
+        Nothing -> traceError "outref not found"
```

Comment:
I guess we need a new error code for this

Reply:
Done in https://github.com/input-output-hk/hydra/commit/116081f9da4ec8179005b675613197350a8f14c8